### PR TITLE
Fixed typo in spelling of definition and VMware

### PR DESCRIPTION
**Pull request by ghost**
This PR is related to https://github.com/openshift/openshift-docs/pull/5607

### CODE REVIEW

**Comment by tmorriso-rh** [comment]
@openshift/team-documentation PTAL.

**Comment by ahardin-rh** [comment]
LGTM!

File: install_config/persistent_storage/dynamically_provisioning_pvs.adoc [hunk 1]
```diff
@@ -108,7 +108,7 @@ plug-ins that are supported. The following sections describe the basic object
 definition for a _StorageClass_ and specific examples for each of the supported
 plug-in types.
 
-[[basic-spec-defintion]]
+[[basic-spec-definition]]
```
Comment (vikram-redhat) [inline install_config/persistent_storage/dynamically_provisioning_pvs.adoc]: @tmorriso-rh - make sure to check that the old anchor tag is not being used in the docs. If it is, then those docs would need changing as well.

**Comment by tmorriso-rh** [comment]
No revision history needed.

File: install_config/persistent_storage/dynamically_provisioning_pvs.adoc (hunk 1 above)
Comment (tmorriso-rh) [reply install_config/persistent_storage/dynamically_provisioning_pvs.adoc]: Yes, I checked these during the build and they are working.

**Pull request merged**